main.cpp

```diff
@@ -0,0 +1,117 @@
+#include <iostream>
+#include <string>
+#include <vector>
+using namespace std;
+
+void print(vector<string> & sent){
+    for(int i = 0; i < sent.size() - 1 ; i++)
+    {
+        cout << sent[i] << " ";
+    }
+    cout << sent[sent.size()-1];
+}
+
+bool check(vector<string>& wordChecks){
+    int index = 0;
+    if(wordChecks[index] != "the" && wordChecks[index] != "The"){
+        cout << "ERROR 1! The sentence '";
+        print(wordChecks);
+        cout << "' does not meet requirements" << endl;
+        return false;
+    }
+
+    index++;
+    if(wordChecks[index] != "lazy" && wordChecks[index] != "smelly")
+    {
+        if(wordChecks[index] != "dog" && wordChecks[index] != "cat")
+        {
+            cout << "ERROR 2! The sentence '";
+            print(wordChecks);
+            cout << "' does not meet requirements" << endl;
+            return false;
+        }
+    } else {
+        index++;
+        while(wordChecks[index] == "lazy" || wordChecks[index] == "smelly"){
```

Comment:
This will index past the end of the vector if the sentence ends with an adjective